### PR TITLE
Added Plexus75 PCB

### PR DESCRIPTION
**Pull request by npspears**
## Description

Plexus75 is available now and some have already shipped out:

purchase link: https://forms.gle/RT6ymQ1QrPEMXjRV9

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/plexus75/config.h [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xCDCD
```
Comment (Erovia) [inline keyboards/plexus75/config.h:21]:
```suggestion
#define VENDOR_ID       0xCDCD
```
Other, VIA-enabled boards already use this VID, so I'd recommend coming up with a unique one.

File: keyboards/plexus75/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+![Plexus75](https://i.imgur.com/c06EAxJ.jpg)
+
+The Plexus75 is a 60% ortholinear keyboard based on the XD75 but instead of an all 1u bottom row the Plexus75 has a more traditional bottom row similar to reg. 60% boards.
+* Keyboard Maintainer: [Nasp](https://github.com/npspears)
+* Hardware Supported: Plexus75 PCB
+* Hardware Availability: https://forms.gle/uSqeNRgJEEQNk4C97
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb plexus75 -km default
```
Comment (Erovia) [inline keyboards/plexus75/readme.md:12]:
```suggestion
    make plexus75:default
```
For now we have the make commands in the readmes.

File: keyboards/plexus75/rules.mk [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# QMK Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```
Comment (Erovia) [inline keyboards/plexus75/rules.mk:11]:
```suggestion
BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
```
I recommend having lite mode, so the users can have an easy reset mode.

**Review by Erovia — CHANGES_REQUESTED** [review]

**Review by npspears — COMMENTED** [review]
Changes applied

**Review by npspears — COMMENTED** [review]
Fixed Vendor ID...again

File: keyboards/plexus75/config.h (hunk 1 above)
Comment (npspears) [reply keyboards/plexus75/config.h]: Changed

File: keyboards/plexus75/readme.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Plexus75
+
+![Plexus75](https://i.imgur.com/c06EAxJ.jpg)
+
+The Plexus75 is a 60% ortholinear keyboard based on the XD75 but instead of an all 1u bottom row the Plexus75 has a more traditional bottom row similar to reg. 60% boards.
+* Keyboard Maintainer: [Nasp](https://github.com/npspears)
```
Comment (fauxpark) [inline keyboards/plexus75/readme.md:6]:
```suggestion

* Keyboard Maintainer: [Nasp](https://github.com/npspears)
```

File: keyboards/plexus75/rules.mk [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = yes        # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/plexus75/rules.mk:22]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
UNICODE_ENABLE = yes        # Unicode
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/plexus75/readme.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Plexus75
+
+![Plexus75](https://i.imgur.com/c06EAxJ.jpg)
```
Comment (noroadsleft) [inline keyboards/plexus75/readme.md:3]: Swaps this image for a smaller version.

```suggestion
![Plexus75](https://i.imgur.com/c06EAxJl.jpg)
```

File: keyboards/plexus75/plexus75.h [hunk 7]
```diff
@@ -0,0 +1,48 @@
+#pragma once
```
Comment (noroadsleft) [inline keyboards/plexus75/plexus75.h:1]: Need a license header here, please.

File: keyboards/plexus75/info.json [hunk 8]
```diff
@@ -0,0 +1,22 @@
+{
+    "keyboard_name": "Plexus75", 
+    "url": "", 
+    "maintainer": "npspears", 
+    "width": 15, 
+    "height": 5, 
+    "layouts": {
+	"default": {
```
Comment (noroadsleft) [inline keyboards/plexus75/info.json:8]: Needs to be an exact match to the name used in the keyboard's header file (`plexus75.h`).

```suggestion
        "LAYOUT_2x2u": {
```

File: keyboards/plexus75/info.json [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+		   },
+        "default_3u": {
```
Comment (noroadsleft) [inline keyboards/plexus75/info.json:11]:
```suggestion
        "LAYOUT_2x3u": {
```

File: keyboards/plexus75/info.json [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+                      },
+        "default_7u": {
```
Comment (noroadsleft) [inline keyboards/plexus75/info.json:14]:
```suggestion
        "LAYOUT_7u": {
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/plexus75/plexus75.h (hunk 7 above)
Comment (npspears) [reply keyboards/plexus75/plexus75.h:1]: added

**Review by fauxpark — APPROVED** [review]

File: keyboards/plexus75/plexus75.h (hunk 7 above)
Comment (noroadsleft) [reply keyboards/plexus75/plexus75.h:1]: `keyboards/plexus75/plexus75.h` still doesn't have a license header.

Comment (npspears) [reply keyboards/plexus75/plexus75.h:18]: sorry, it should be there now

**Review by npspears — COMMENTED** [review]
all changes requested have been made

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**